contrib/babelfishpg_tsql/src/catalog.c

```diff
@@ -1298,14 +1331,19 @@ clean_up_bbf_server_def()
 {
 	char 			*query_str;
 	StringInfoData 	query;
+	int save_nestlevel;
 
 	initStringInfo(&query);
 
 	appendStringInfo(&query, "TRUNCATE TABLE sys.babelfish_server_options CASCADE");
```

Comment:
when do we call this? do we have a test case for this?

Reply:
It is called during DROP EXTENSION tds_fdw, we already have tests for this like BABEL-4168.